Linker/Linker.Android/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.companyname">
-    <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
-    <application android:label="Linker.Android"></application>
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          android:versionCode="1"
+          android:versionName="1.0"
+          package="com.companyname">
+  <uses-sdk android:minSdkVersion="21"
+            android:targetSdkVersion="28" />
+  <application android:label="Linker.Android"></application>
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
can you add a comment on why it's needed?

Reply:
Added